Cognigy/chat-components
# Do not collate message if prev message is rating

The spacing below the Feedback submitted status pill is incorrect because of collation. So, this PR prevents message collation when the previous message is a rating feedback.

NOTE: Test this PR along with https://github.com/Cognigy/WebchatWidget/pull/363 or after that PR is merged.
How to test:
- Build & Pack the chat-components folder
- install the package in webchat widget repo
- Use an endpoint pointing to a flow with request rating node or use this endpoint:  https://endpoint-dev.cognigy.ai/a7f6316334debd965e4668ec402043e3498c69f0146fa4b96f29c059337eeab7
- Chat with the bot and provide a rating when prompted.
- After submitting the feedback 'Feedback submitted' status pill will appear in the history
- Now send another user message.
- The spacing between the pill and the following user message should be according to figma (Refer to User joined and Meeting confirmed status pills)

BEFORE
![before](https://github.com/Cognigy/chat-components/assets/29442479/adfa2907-afa5-4b70-972b-5533633279b5)

AFTER
![after](https://github.com/Cognigy/chat-components/assets/29442479/c07f8444-8534-4386-adb4-1f6202e991c0)
